### ISSUE TITLE
Disable rules inline using gherkin comments

### ISSUE THREAD

**Issue by theRealAlpaca** [vsiakka/gherkin-lint#268]
Having an option to ignore certain files is great but it's quite limiting because it ignores all lining rules for certain files/directories that match the pattern. I believe there should be an option similar to `eslint-disable` in ESLint (https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments) or like `nolint` in `golangci-lint` (https://golangci-lint.run/usage/false-positives/#nolint-directive).

I'd suggest the following format (the naming can be adjusted)
```gherkin
# gherkin-ignore: name-length
Given I do super long step that exceeds the provided step max length there is an option to ignore it
```
